spec/spec_helper.rb

```diff
@@ -51,12 +51,6 @@
   #     --seed 1234
   config.order = 'random'
 
-  # Setting up DB cleaning to maintain empty rows
```

Comment:
Nothing really. I was going through spec_helper.rb and found this, hence I removed it as well. Should I send a different PR with just this change?